src/runtime/test/test-environment-test.ts

```diff
@@ -1,13 +1,13 @@
-import {registerSystemExceptionHandler} from '../arc-exceptions.js';
+import { registerSystemExceptionHandler } from '../arc-exceptions.js';
```

Comment:
The spaces around `registerSystemExceptionHandler` don't match what we do in src/runtime

Reply:
done